markup of the pages of the payment method

MasatakaOba opened the pull request:
## WHAT
I made the pages of the payment method.
1. Add haml files to register payment style
2. Add scss files for pages' decration.
3. edit routes

## WHY
Users will be able to register the information of payment method at those pages.

### The links of each pages is below
/card ->button to start to register
<img width="784" alt="image" src="https://user-images.githubusercontent.com/27143270/54874269-125a9f80-4e2b-11e9-9e0e-5a57a7be9202.png">

/card/new -> input page
<img width="594" alt="image" src="https://user-images.githubusercontent.com/27143270/54874275-21d9e880-4e2b-11e9-82d8-9d2a873268df.png">

/card/show -> show information which is registered
<img width="737" alt="image" src="https://user-images.githubusercontent.com/27143270/54874284-328a5e80-4e2b-11e9-98ad-42fe14f91539.png">

Commit:
markup of the pages of the information of how to pay

1. Add haml files to register payment style
2. Add scss files for pages' decration.
3. edit routes
the links of each pages is below
/card
->button to start to register
/card/new
-> input page
/card/show
-> show information which is registered

hiroki5123 commented on app/views/card/show.html.haml:
```diff
@@ -0,0 +1,37 @@
+          %label クレジットカード一覧
+          .card-flexbox
+            .add-card-settings-left
+              =image_tag "card-icon-visa.svg", alt: "visa", size: "34x20"
```
sizeはCSS側に指定しましょう

MasatakaOba replied:
こちら対応しました。

hiroki5123 commented on app/views/card/new.html.haml:
```diff
@@ -0,0 +1,59 @@
+          %span.form-require 必須
+          = text_field_tag "card-id", "", class: "input-default", placeholder: "半角数字のみ"
+          .card-image-list
+            = image_tag "card-icon-visa.svg", alt: "visa", size: "49x20"
```
sizeはCSS側に指定しましょう

MasatakaOba replied:
こちら対応しました。

hiroki5123 commented on app/views/card/new.html.haml:
```diff
@@ -0,0 +1,59 @@
+    %li
+      %span.mercari
+        = link_to "メルカリ", "#", class: "underline-opacity"
+      = fa_icon "angle-right"
```
これだとアイコンをクリックした時に反応しない気がするのですが大丈夫ですか？

MasatakaOba replied:
rootなど存在するものについては追記しました。
まだ完成していない部分があるので、そちらについてはでき次第パスを加えていきます。

hiroki5123 replied:
一応このコメントはアイコンをlink_to 〜 doで囲わなくていいのかという意図のコメントです。

MasatakaOba replied:
失礼しました。
このアイコンは本家のサイトのリンクには含まれていなかったため、doで囲いませんでした。

hiroki5123 replied:
承知しました！

Commit:
Move the code from HTML(haml) to CSS(scss) / Add paths on the pages

1. Use not html but css to apply the size of the logo of credit cards
2. Add some paths which have already existed to the pages

MasatakaOba commented:
> LGTMです！

ありがとうございます！
